ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -76,25 +69,6 @@ import { PanelContent } from './runtime-configuration-panel/utils';
 
 const { useState, useEffect } = React;
 
-interface DeriveCurrentRuntimeProps {
-  crFromCustomRuntimeHook: Runtime;
-  gcePersistentDisk: Disk;
-}
-export const deriveCurrentRuntime = ({
```

Comment:
This logic was used for deriving the current runtime if there was a deleted one available. This is no longer possible.